### PR TITLE
Bug 1204054 - Adding FTU to homescreen when making for Raptor tests

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e68e84dedfb9e358c73f3f93ead6e076cbe64d84

File: apps/ftu/build/build.js [hunk 1]
```diff
@@ -16,11 +16,17 @@ FtuAppBuilder.prototype.setOptions = function(options) {
...
   this.gaia.stageDir = this.stageDir;
   this.gaia.gaiaDir = options.GAIA_DIR;
+  this.raptorMode = options.RAPTOR === '1';
 };
 
 FtuAppBuilder.prototype.generateManifest = function() {
   var manifestObject = importBuild.generateManifest(this.webapp, this.gaia);
   var file = utils.getFile(this.stageDir.path, 'manifest.webapp');
+
+  if (this.raptorMode) {
```
Comment (sfoster) [inline apps/ftu/build/build.js]: Add short comment like 'remove role:system so FTU gets a homescreen icon to allow raptor/marionette to launch it' as its a bit indirect and non-obvious

Comment (eliperelman) [inline apps/ftu/build/build.js]: Good idea, will do.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3e5542d315af1ef395f9d96969e29ab410e800fe

**Pull request merged**